Makefile

```diff
@@ -56,22 +60,21 @@ dist/apple-touch-icon.png: dist/icon-180x180.png | dist
 	$(INSTALL) -m 0664 $< $@
 dist/favicon.ico: dist/icon-32x32.png | dist
 	$(CONVERT) $^ $@
-dist/main.css: www/main.scss node_modules | dist
-	$(SASSC) --style compressed $< $@
-dist/main.js: www/app.js node_modules | dist
-	$(NODE) $(word 2,$^)/webpack/bin/webpack.js --mode=production ./$< -o $(dir $@)
 dist/%: www/% | dist
 	$(INSTALL) -m 0664 $< $@
 
+
 .PHONY: www
 www: \
 	dist/icon.svg \
-	dist/main.css \
-	dist/main.js \
 	dist/CNAME \
 	dist/favicon.ico \
 	dist/icon-192x192.png \
  	dist/icon-512x512.png \
 	dist/apple-touch-icon.png \
-	dist/index.html \
 	dist/manifest.webmanifest
+	webpack
```

Comment:
This doesn't look valid. The `www` is meant to build the website for release. By appending the `website` target here, you are running it on development mode too, which is not intended. In any case, you missed the `\` at the end, so its not even applying correctly.